compiler/src/iree/compiler/Codegen/Common/ReconcileTranslationInfo.cpp

```diff
@@ -360,8 +361,8 @@ void ReconcileTranslationInfoPass::runOnOperation() {
 
   auto exportOps = variantOp.getOps<IREE::HAL::ExecutableExportOp>();
   if (!llvm::hasSingleElement(exportOps)) {
-    variantOp.emitOpError("reconciliation for multiple export ops unsupported");
-    return signalPassFailure();
+    variantOp.emitWarning("reconciliation for multiple export ops unsupported");
```

Comment:
This might just result in unnecessary warning skew. Its ok to drop the warning and just return.